Sources/Model/Transfer/HyperwalletTransfer.swift

```diff
@@ -0,0 +1,202 @@
+        }
+
+        /// Sets the transfer destination amount.
+        ///
+        /// - Parameter destinationAmount: The payment amount in the specified currency loaded into
+        ///                                your merchant account.
+        /// - Returns: a self `HyperwalletTransfer.Builder` instance.
+        public func destinationAmount(_ destinationAmount: String?) -> Builder {
+            self.destinationAmount = destinationAmount
+            return self
+        }
+
+        /// Sets the transfer destination currency.
+        ///
+        /// - Parameter destinationCurrency: The currency of the payment amount loaded into your merchant
+        ///                                  account in ISO 4217 format.
+        /// - Returns: a self `HyperwalletTransfer.Builder` instance.
+        public func destinationCurrency(_ destinationCurrency: String?) -> Builder {
+            self.destinationCurrency = destinationCurrency
+            return self
+        }
+
+        /// Sets the transfer memo.
+        ///
+        /// - Parameter memo: An internal memo for the SpendBack transfer (will not be visible to
+        ///                                  the user making the payment).
+        /// - Returns: a self `HyperwalletTransfer.Builder` instance.
+        public func memo(_ memo: String?) -> Builder {
+            self.memo = memo
+            return self
+        }
+
+        /// Sets the transfer notes.
+        ///
+        /// - Parameter notes: A description for the SpendBack transfer.
+        /// - Returns: a self `HyperwalletTransfer.Builder` instance.
+        public func notes(_ notes: String?) -> Builder {
+            self.notes = notes
+            return self
+        }
+
+        /// Sets the transfer source amount.
+        ///
+        /// - Parameter sourceAmount: The payment amount in the specified currency that is debited
+        ///                           from the sourceToken.
+        /// - Returns: a self `HyperwalletTransfer.Builder` instance.
+        public func sourceAmount(_ sourceAmount: String?) -> Builder {
+            self.sourceAmount = sourceAmount
+            return self
+        }
+
+        /// Sets the transfer source currency.
+        ///
+        /// - Parameter sourceCurrency: The currency of the payment amount debited from the sourceToken
+        ///                             in ISO 4217 format.
+        /// - Returns: a self `HyperwalletTransfer.Builder` instance.
+        public func sourceCurrency(_ sourceCurrency: String?) -> Builder {
+            self.sourceCurrency = sourceCurrency
+            return self
+        }
+
+        // Builds a new instance of the `HyperwalletTransfer`.
+        ///
+        /// - Returns: a new instance of the `HyperwalletTransfer`.
+        public func build() -> HyperwalletTransfer {
+            return HyperwalletTransfer(clientTransferId: clientTransferId,
+                                       createdOn: nil,
```

Comment:
default value in init is nil so no need to add createdOn or any other nil values here.

Reply:
Fixed